inference-engine/src/mkldnn_plugin/nodes/mkldnn_pooling_node.h

```diff
@@ -28,11 +28,27 @@ class MKLDNNPoolingNode : public MKLDNNNode {
         return false;
     }
 
+    void prepareParams() override;;
+    void executeDynamicImpl(mkldnn::stream strm) override { execute(strm); }
+
     static bool isSupportedOperation(const std::shared_ptr<const ngraph::Node>& op, std::string& errorMessage) noexcept;
 
+protected:
+    AttrPtr initPrimitiveAttr() const override;
+
 private:
-    void setPostOps(mkldnn::primitive_attr &attr, bool initWeights = false);
+    void setPostOps(mkldnn::primitive_attr &attr, bool initWeights = false) const;
+
+    static constexpr Dim shapeDymmyVal = 64;
```

Comment:
Why did you introduce this constant? It is pretty the same as the default value in the `makeDummyShape` call.

Reply:
I thought it was better to be sure that `makeDummyDesc` and `makeDummyShape` are called everywhere with the same `dummyValue`
but looks like that default value enough